xray/formats/table.go

```diff
@@ -5,40 +5,75 @@ package formats
 }
 
-type LicenseTableRow struct {
+type vulnerabilityScanTableRow struct {
+	Severity string `col-name:"Severity"`
+	// For sorting
+	SeverityNumValue       int
+	DirectPackages         []directPackagesTableRow `embed-table:"true"`
+	ImpactedPackageName    string                   `col-name:"Impacted\nPackage\nName"`
+	ImpactedPackageVersion string                   `col-name:"Impacted\nPackage\nVersion"`
+	FixedVersions          string                   `col-name:"Fixed\nVersions"`
+	ImpactedPackageType    string                   `col-name:"Type"`
+	Cves                   []cveTableRow            `embed-table:"true"`
+	IssueId                string                   `col-name:"Issue ID" extended:"true"`
+}
+
+type licenseTableRow struct {
 	LicenseKey                string                       `col-name:"License"`
-	DirectDependencies        []DirectDependenciesTableRow `embed-table:"true"`
+	DirectDependencies        []directDependenciesTableRow `embed-table:"true"`
 	ImpactedDependencyName    string                       `col-name:"Impacted\nDependency"`
 	ImpactedDependencyVersion string                       `col-name:"Impacted\nDependency\nVersion"`
 	ImpactedDependencyType    string                       `col-name:"Type"`
 }
 
-type LicenseViolationTableRow struct {
-	LicenseKey                string                       `col-name:"License"`
-	Severity                  string                       `col-name:"Severity"`
-	SeverityNumValue          int                          // For sorting
-	DirectDependencies        []DirectDependenciesTableRow `embed-table:"true"`
+type licenseScanTableRow struct {
+	LicenseKey             string                   `col-name:"License"`
+	DirectDependencies     []directPackagesTableRow `embed-table:"true"`
+	ImpactedPackageName    string                   `col-name:"Impacted\nPackage"`
+	ImpactedPackageVersion string                   `col-name:"Impacted\nPackage\nVersion"`
+	ImpactedDependencyType string                   `col-name:"Type"`
+}
+
+type licenseViolationTableRow struct {
+	LicenseKey string `col-name:"License"`
+	Severity   string `col-name:"Severity"`
+	// For sorting
+	SeverityNumValue          int
+	DirectDependencies        []directDependenciesTableRow `embed-table:"true"`
 	ImpactedDependencyName    string                       `col-name:"Impacted\nDependency"`
 	ImpactedDependencyVersion string                       `col-name:"Impacted\nDependency\nVersion"`
 	ImpactedDependencyType    string                       `col-name:"Type"`
 }
 
-type OperationalRiskViolationTableRow struct {
-	Severity                  string                       `col-name:"Severity"`
-	SeverityNumValue          int                          // For sorting
-	DirectDependencies        []DirectDependenciesTableRow `embed-table:"true"`
+type licenseViolationScanTableRow struct {
+	LicenseKey string `col-name:"License"`
```

Comment:
It looks like `LicenseKey` can also be made private. I suggest we go over all the other fields in this file and checl wether they can be made private as well.